### PR TITLE
next-bg-image integration

### PR DESCRIPTION
**Pull request by jaredh159**
see #113 for explanation of 3rd commit

### CODE REVIEW

File: apps/cli/package.json [hunk 1]
```diff
@@ -52,7 +52,7 @@
     "puppeteer": "^13.5.0",
     "qs": "^6.9.1",
     "serve": "^11.2.0",
-    "sharp": "^0.28.3",
+    "sharp": "^0.33.0",
```
Comment (jaredh159) [inline apps/cli/package.json:55]: was getting a build error working on pnpm stuff, this fixed it. sharp is the death of me.

File: apps/evans/components/404/NotFoundHeroBlock.tsx [hunk 2]
```diff
@@ -1,53 +1,51 @@
...
 
 const NotFoundHeroBlock: React.FC = () => (
-  <BackgroundImage
-    src={SheepBg}
-    fit="cover"
-    position="[object-position:center_80%]"
-    className="text-center text-white"
+  <NextBgImage
+    className="text-center text-white px-10 py-20 sm:px-16 sm:py-24 md:py-24 xl:py-32"
+    src={[bgColor(`rgba(0, 0, 0, 0.38)`), SheepBg]}
```
Comment (jaredh159) [inline apps/evans/components/404/NotFoundHeroBlock.tsx:12]: here's an example of using the `bgColor` helper component from `next-bg-image`

File: apps/evans/components/pages/friend/BookByFriend.tsx [hunk 3]
```diff
@@ -46,7 +46,7 @@ const BookByFriend: React.FC<Props> = (props) => {
         {...props}
         scaler={isAlone ? 1 / 2 : 1 / 3}
         scope={isAlone ? `1-2` : `1-3`}
-        className={isAlone ? `-mt-20 md:mt-4 md:-ml-32` : `mt-4 md:-ml-24`}
+        className={cx(`mt-4`, isAlone ? `md:mt-4 md:-ml-32` : `md:-ml-24`)}
```
Comment (jaredh159) [inline apps/evans/components/pages/friend/BookByFriend.tsx:49]: see #113

File: apps/evans/lib/color.ts [hunk 4]
```diff
@@ -0,0 +1,36 @@
+import { MAROON_RGB, GOLD_RGB, BLUE_RGB, BLACK_RGB } from '@friends-library/theme';
+import { LANG } from '@/lib/env';
+
+export function bgLayer(
```
Comment (jaredh159) [inline apps/evans/lib/color.ts:4]: this file/function i pulled over from the old gatsby dir, a helper for creating bg gradient layers that i used with gatsby-bg-image, and works well with our component as well.

File: apps/evans/package.json [hunk 5]
```diff
@@ -21,6 +21,7 @@
     "autoprefixer": "10.4.14",
     "js-cookie": "^2.2.1",
     "next": "13.4.12",
+    "next-bg-image": "0.1.0-beta.8",
```
Comment (jaredh159) [inline apps/evans/package.json:24]: 🎉

File: apps/evans/pages/_document.tsx [hunk 6]
```diff
@@ -1,9 +1,12 @@
 import React from 'react';
 import { Html, Head, Main, NextScript } from 'next/document';
+import { NextBgStaticCss } from 'next-bg-image';
 
 const Document: React.FC = () => (
   <Html lang="en">
-    <Head></Head>
+    <Head>
+      <NextBgStaticCss />
```
Comment (jaredh159) [inline apps/evans/pages/_document.tsx:8]: this is what i mentioned in slack, for the time being we have to put this component in the dom once.

File: apps/evans/components/404/NotFoundHeroBlock.tsx (hunk 2 above)
Comment (kiahjh) [reply apps/evans/components/404/NotFoundHeroBlock.tsx:12]: Yeah that's really nice 👍

**Pull request merged**